cfgov/unprocessed/apps/filing-instruction-guide/css/main.less

```diff
@@ -4,10 +4,16 @@
 .o-fig {
   // Desktop
   .respond-to-min( @bp-med-min, {
-    // Expand it to full height on desktop with optional scrolling
-    .o-fig_sidebar {
+    // Expand the sticky nav to full height on desktop with optional scrolling
+    .u-layout-grid_secondary-nav__sticky {
```

Comment:
So we create this modifier in layout-1-3.less and then customize it inside the FIG … … no 

![elcaminono](https://github.com/cfpb/consumerfinance.gov/assets/704760/7b6b891e-3f96-4c87-b0f2-eb41df96403b)

Reply:
![standards](https://media.tenor.com/QcB6Xaba6sMAAAAj/do-i-have-standards-ryanfluffbruce.gif)